tools/uart_hw_test/README.md

```diff
@@ -40,24 +40,90 @@ It is MANDATORY to first run the uart_hw_test tash command and enter the port nu
 After this run the ./uart_test.sh test app on linux and enter the port number.
 If this sequence is not followed, then the result is not proper.
 
+
+Below is a sample set of results for linux case :
+```
+Enter UART port number : 1
+Opened UART port : /dev/ttyUSB1
+######################## UART Test Start #############################
+######################################################################
+Test Scenario 0: UART test only
+######################################################################
+[BAUD 115200 Tx] PASS
+[BAUD 115200 Rx] PASS
+[BAUD  38400 Tx] PASS
+[BAUD  38400 Rx] PASS
+[BAUD  19200 Tx] PASS
+[BAUD  19200 Rx] PASS
+[BAUD   9600 Tx] PASS
+[BAUD   9600 Rx] PASS
+[BAUD   4800 Tx] PASS
+[BAUD   4800 Rx] PASS
+
+######################################################################
+Test Scenario 1: UART test with background FS ops
+######################################################################
+[BAUD 115200 Tx] PASS
+[BAUD 115200 Rx] PASS
+[BAUD  38400 Tx] PASS
+[BAUD  38400 Rx] PASS
+[BAUD  19200 Tx] PASS
+[BAUD  19200 Rx] PASS
+[BAUD   9600 Tx] PASS
+[BAUD   9600 Rx] PASS
+[BAUD   4800 Tx] PASS
+[BAUD   4800 Rx] PASS
+
+######################################################################
+Test Scenario 2: UART test with background console logs
+######################################################################
+[BAUD 115200 Tx] PASS
+[BAUD 115200 Rx] PASS
+[BAUD  38400 Tx] PASS
+[BAUD  38400 Rx] PASS
+[BAUD  19200 Tx] PASS
+[BAUD  19200 Rx] PASS
+[BAUD   9600 Tx] PASS
+[BAUD   9600 Rx] PASS
+[BAUD   4800 Tx] PASS
+[BAUD   4800 Rx] PASS
+
+######################################################################
+Test Scenario 3: UART test with background iperf
+######################################################################
+[BAUD 115200 Tx] PASS
+[BAUD 115200 Rx] PASS
+[BAUD  38400 Tx] PASS
+[BAUD  38400 Rx] PASS
+[BAUD  19200 Tx] PASS
+[BAUD  19200 Rx] PASS
+[BAUD   9600 Tx] PASS
+[BAUD   9600 Rx] PASS
+[BAUD   4800 Tx] PASS
+[BAUD   4800 Rx] PASS
+######################################################################
+################## UART Test End [PASS: 40, FAIL: 0] #################
+```
+
```

Comment:
I think the testcase result should not be added here.
It should be added as a comment by the author for this PR to show the effective functioning of the newly implemented/updated test case.

Reply:
It was like this before my commit also , so i kept it like this only .